ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -625,6 +626,7 @@
       id: 'BA AV Eidos Relative Virtue Umbral',
       regex: Regexes.gameLog({ line: 'Relative Virtue gains the effect of Umbral Essence.*?', capture: false }),
       regexDe: Regexes.gameLog({ line: 'Die Relative Tugend erhält den Effekt von.*?Arm der Dunkelheit.*?', capture: false }),
+      regexFr: Regexes.gameLog({ line: 'Vertu relative bénéficie de l\'effet  Bras de Ténèbres.*?', capture: false }),
```

Comment:
Same questions

Reply:
I would suggest doing what I did for German and making them `Vertu relative bénéficie de l\'effet.*?Bras de Ténèbres.*?`

Reply:
It is done, thank you.